lib/conncache.c

```diff
@@ -689,7 +689,7 @@ static void connc_discard_conn(struct conncache *connc,
    * not what we want. */
   if(aborted)
     done = TRUE;
-  else if(!done) {
+  else {
```

Comment:
I think this should be `if(!done) {` since we do not run the shutdown for aborted connections.

Reply:
fair!